test/values.ts

```diff
@@ -509,6 +509,20 @@ export const mockTokenBalances: Record<string, StoredTokenBalance> = {
     priceFetchedAt: Date.now(),
     isCashInEligible: true,
   },
+  [mockEthTokenId]: {
```

Comment:
I've been hesitant to add this in many prior multichain PRs thinking it may cause some unexpected test failures and instead just creating an eth token balance entry in multiple test files, but looks like things are fine 😄

Reply:
lol yeah I was wondering if it would break stuff. It did at first, but I was able to fix just by making sure the balance was not the highest of the other balances here